disk.go

```diff
@@ -42,8 +87,22 @@ func (d *Disk) scheduleWrite(sizebytes int) time.Duration {
 	at := sizeToDuration(sizebytes, "B", int64(d.MBps), "MB")
 	w := d.writes
 
+	// Simulate adding i/o to the queue.
+	d.qMutex.Lock()
+	d.pendingwbytes += int64(sizebytes)
+	qSz := float64(d.pendingwbytes) / float64(configStorage.sizeDataChunk * 1024)
+	d.qMutex.Unlock()
```

Comment:
It's a bit unusual to measure queue depth in bytes. Typically, a number of pending I/Os, no matter what size they are.